frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -536,9 +539,9 @@ function renderAllReceiverRows(props: {
             >
                 <Grid className={`title-column ${titleClassName}`}>
                     <div className={"title-text"}>
-                        {orgName}
+                        <Link to={linkOrgSettings}>{orgName}</Link>
```

Comment:
Looks like the use of `Link` is causing some build error...perhaps a test needs to be wrapped in `renderWithRouter`?

Reply:
Thanks. Somehow I didn't spot that issue. Fixed it and the checks are rerunning.